web/src/info_overlay.js

```diff
@@ -120,6 +130,12 @@ def zulip():
         markdown: "/me is busy working",
         output_html: '<p><span class="sender_name-in-status">Iago</span> is busy working</p>',
     },
+    {
+        markdown: "<time:2023-05-28T13:30:00+05:30>",
+        output_html:
+            '<p><time datetime="2023-05-28T08:00:00Z"><i class="fa fa-clock-o"></i>Sun, May 28, 2023, 1:30 PM</time></p>',
+        effect_html: "(displayed to everyone in their own time zone)",
+    },
```

Comment:
I think after seeing this, let's skip the `effect_html`; you get the same information by hovering over it with the mouse, and it feels pretty ugly with a wrapped line there.